gradle/dependencies.gradle

```diff
@@ -142,8 +142,8 @@ versions += [
   slf4j: "1.7.36",
   snappy: "1.1.10.1",
   spotbugs: "4.7.3",
-  swaggerAnnotations: "2.2.8",
-  swaggerJaxrs2: "2.2.8",
+  swaggerAnnotations: "2.2.14",
```

Comment:
can't update this. Versions after 2.2.8 require minimum JDK 11 - https://github.com/swagger-api/swagger-core/pull/4377

Please also add a comment about this here for future folks who may want to change this

Reply:
Thanks for review! I should have took it more carefully!
Will fix it.